python/dstat-kudu/kudu_dstat.py

```diff
@@ -25,23 +21,27 @@ def open_or_create_table(client, table, drop=False):
 
   if not exists:
     # Create the schema for the table, basically all float cols
-    cols = [kudu.ColumnSchema.create("ts", kudu.INT64)]
-    cols += [kudu.ColumnSchema.create(x, kudu.FLOAT) for x in DSTAT_COL_NAMES]
+    builder = kudu.schema_builder()
+    builder.add_column("ts", kudu.int64, nullable=False, primary_key=True)
+    [builder.add_column(col, kudu.float_) for col in DSTAT_COL_NAMES]
+    schema = builder.build()
+
+    # Create hash partitioning buckets
+    partitioning = Partitioning().add_hash_partitions('ts', 2)
 
-    # Based on the column meta data create a new schema object, where the first column
-    # is the key column.
-    schema = kudu.schema_from_list(cols, 1)
-    client.create_table(table, schema)
+    # Create table with a replication factor of 1 in case examples are run
+    # on a machine with a single tablet server
```

Comment:
hrm, this bit about replication factor 1 doesn't correspond to the line of code below it